drivers/sensor/ti/ina23x/ina230.c

```diff
@@ -77,26 +101,33 @@ static int ina230_sample_fetch(const struct device *dev, enum sensor_channel cha
 		return -ENOTSUP;
 	}
 
+	/* Run the ADC to get new data */
+	ret = ina230_adc_run(dev);
+	if (ret < 0) {
+		LOG_ERR("Failed to run sample");
+		return ret;
+	}
+
 	if ((chan == SENSOR_CHAN_ALL) || (chan == SENSOR_CHAN_VOLTAGE)) {
 		ret = ina23x_reg_read_16(&config->bus, INA230_REG_BUS_VOLT, &data->bus_voltage);
 		if (ret < 0) {
-			LOG_ERR("Failed to read bus voltage");
+			LOG_ERR("Failed to read %s", "bus voltage");
```

Comment:
Why use formatting here? This is unnecessarily expensive

Reply:
Using the same format string and only changing the argument allows the format string to reused for all log instances, saving ROM. It is a somewhat common pattern in Zephyr.

Reply:
Interesting, this might be something we bring up with the TSC, we use tokenized logging so each string regardless of length only costs us 4 bytes, using string formats adds runtime costs and ROM. @keith-zephyr what do you think?

Reply:
Last time I looked at that feature #42840 hadn't been implemented, which sort of defeated the point.
In other PR's I have been requested to use the form I used here, but I see why its unhelpful for dictionary logging.